telegram_handler.rb

```diff
@@ -115,7 +128,49 @@ def self.handle_input(bot, message)
         #   bot.api.send_message(chat_id: message.chat.id, text: "Number of months tracked: #{number_of_months.round(1)}")
         #   bot.api.send_message(chat_id: message.chat.id, text: "Averaging #{average_number_of_moods.round(1)} per day")
         when "/start"
+          kb=$kb;
```

Comment:
Similar keyboard code is repeated multiple times.
How about we encapsulate it in a function that receives a chat_id and returns the keyboard object?